beacon-chain/sync/subscriber_beacon_blocks.go

```diff
@@ -81,19 +81,19 @@ func (s *Service) reconstructAndBroadcastBlobs(ctx context.Context, block interf
 	if s.cfg.blobStorage == nil {
 		return
 	}
-	indices, err := s.cfg.blobStorage.Indices(blockRoot, block.Block().Slot())
+	summary := s.cfg.blobStorage.Summary(blockRoot)
+	cmts, err := block.Block().Body().BlobKzgCommitments()
 	if err != nil {
-		log.WithError(err).Error("Failed to retrieve indices for block")
-		return
+		log.WithError(err).Error("Failed to read commitments from block")
```

Comment:
```suggestion
		log.WithError(err).Error("Failed to read commitments from block")
		return
```

Seems like a logic change? Why proceed when there is an error?

Reply:
This change was a little odd because the function you are commenting on no longer needs to check commitments, except for the purpose of incrementing this log. If retrieving `cmts` generates an error here, it will also fail when calling `ReconstructBlobSidecars` and failing there will result in an error that makes the most sense to me rolled up with the other errors from that function `Failed to reconstruct blob sidecars`. So the behavior is 2 logs instead of 1, but otherwise same logic. 

But I agree with your suggestion, it's weird to not short circuit here and could be a footgun if code is inserted in between those calls. I'll change it.